app/data/models/vet-visit.js

```diff
@@ -0,0 +1,35 @@
+const createReference = require('../../lib/create-reference')
+module.exports = (sequelize, DataTypes) => {
+  return sequelize.define('vetVisit', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      autoIncrement: true,
+      defaultValue: sequelize.UUIDV4
+    },
+    reference: {
+      type: DataTypes.STRING,
+      set (val) {
+        this.setDataValue('reference', val.toUpperCase())
+      }
+    },
+    applicationReference: DataTypes.STRING,
```

Comment:
I think it would be worth making this a foreign key to `application.reference`, with a one-2-one relation. What do you think?